João Schimmelpfeng

joaoschimmelpfeng opened the pull request:
Bronze.

joaoschimmelpfeng committed:
Bronze

Bronze

joaoschimmelpfeng committed:
Adicionado preço

Precoprecopreco.

joaoschimmelpfeng committed:
Localizable String

String

joaoschimmelpfeng committed:
Primeiro topico do Prata.

For the silver.

joaoschimmelpfeng commented:
Bronze concluido E Prata 1

ViniciusMiana commented on iTunesSearch.xcodeproj/project.pbxproj:
```diff
@@ -312,7 +345,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Porque voce trocou o target para 8.0?

joaoschimmelpfeng replied:
Troquei pois precisava rodar o simulador enquanto eu ainda estava baixando a nova versão do xcode.

ViniciusMiana commented on iTunesSearch/TableViewController.m:
```diff
@@ -55,14 +56,26 @@ -(UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NS
-    [celula.tipo setText:@"Filme"];
+    [celula.preco setText:[filme.preco stringValue]];
+    [celula.tipo setText:NSLocalizedString(filme.tipo, nil)];
+    [celula setViewTumb:filme.artWork];
```
Vc esta trazendo muito mais dados que isso. Precisa mostrar mais informações para ganhar o Bronze1

joaoschimmelpfeng replied:
Adicionei imagem agora.

ViniciusMiana commented on pt.lproj/Localizable.strings:
```diff
@@ -0,0 +1,8 @@
+  Created by João Vitor dos Santos Schimmelpfeng on 10/03/15.
+  Copyright (c) 2015 joaquim. All rights reserved.
+*/
+"feature-movie" = "Filme";
```
Muito bom! Falta colocar estas Strings para os outros idiomas. Este desafio vai ser o Prata-n, portanto vale a pena ir adiantando.

ViniciusMiana commented on iTunesSearch/TableViewController.m:
```diff
@@ -27,11 +28,11 @@ - (void)viewDidLoad {
+    [searchBar1 setDelegate:self];
+    
+    itunes = [iTunesManager sharedInstance];
+    midias = [itunes buscarMidias:@"Alien"];
```
So esta buscando Alien, porque sera?

joaoschimmelpfeng replied:
Agora deixei em branco, para que não liste nada até que o usuário digite algo na area pesquisar e pressione o botão de pesquisa.

joaoschimmelpfeng committed:
Bronze Refeito

Refeito.

joaoschimmelpfeng committed:
Pesquisa dividida por sections.

Pesquisa dividida por sections.
Navigation bar adicionado, mas ainda não funcional.

joaoschimmelpfeng committed:
Desafio do Regex, do navbar completos

Detalhes do regex. se voce digitar -music- ou -movie- ele procura só
por este tipo.

joaoschimmelpfeng committed:
Adicionado persistencia.

Ele salva o ultimo termo buscado então ao abrir o aplicativo pela
segunda vez, ja estará automaticamente no searchbar o ultimo termo
procurado.

joaoschimmelpfeng committed:
Adicionado icones conforme categoria

Agora os objetos das categorias possuem um ícone referente sua
categoria.

joaoschimmelpfeng commented:
Todos os desafios.
